VoxelSniperSpigot/src/main/java/com/github/kevindagame/voxelsniper/bstats/BrushUsersCounter.kt

```diff
@@ -0,0 +1,26 @@
+package com.github.kevindagame.voxelsniper.bstats
+
+import com.github.kevindagame.voxelsniper.events.player.PlayerSnipeEvent
+import java.util.*
+
+class BrushUsersCounter {
+    fun registerListeners() {
+        PlayerSnipeEvent.registerListener { event: PlayerSnipeEvent -> onBrushUse(event) }
```

Comment:
```suggestion
        PlayerSnipeEvent.registerListener(this::onBrushUse)
```

Reply:
Die change vind kotlin niet leuk. Ik laat het zo

Reply:
seemed to work fine for me..

Reply:
![image](https://user-images.githubusercontent.com/65958288/211159293-290c2c0c-6cb2-4bac-86d1-d3b4ce52bb0c.png)

Reply:
nvm zie al wat er mis is

Reply:
haakjes